src/ui/base-page-storyBrowse.js

```diff
@@ -17,7 +17,9 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     fluid.defaults("sjrk.storyTelling.base.page.storyBrowse", {
         gradeNames: ["sjrk.storyTelling.base.page"],
         model: {
-            storyBrowseDisplayPreference: ""
+            persistedValues: {
```

Comment:
Could you add a code comment to explain:
1. what will be tracked in the model;
2. what will be tracked in `model.persistedValues`;
3. the goal of having `model.persistedValues`.

Reply:
I've left a comment in `base-page.js` which outlines the use of that collection, but I'll add these clarifications

Reply:
What do you think of the following, if it's on the `base-page` grade?
> // only values in this colleciton will be persisted by the cookieStore
> // the goal of separating them is to allow the use of other model values
> // such as the view state values in `sjrk.storyTelling.base.page.storyEdit`
> // without saving them.

Reply:
Yes, it's ok to have the comment on the `base-page` grade. It would be better if you could also write about what will be tracked in this collection for persistence. Thanks.

Reply:
Okay, here's a revised version of that comment:

> // Only values in this colleciton will be persisted by the cookieStore.
> //
> // Currently, those values are the `uiLanguage` in this grade and
> // `storyBrowseDisplayPreference` in `sjrk.storyTelling.base.page.storyBrowse`.
> //
> // The goal of separating them is to allow the use of other model values
> // such as the view state values in `sjrk.storyTelling.base.page.storyEdit`
> // without saving them.

Reply:
Looks great. Thanks.